L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -556,6 +566,9 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
   // * produce the GlobalObjectMapRecord
   std::unique_ptr<GlobalObjectMapRecord> gtObjectMapRecord(new GlobalObjectMapRecord());
 
+  // if (m_produceAXOL1TLScore)
+  std::unique_ptr<AXOL1TLScoreBxCollection> uGtAXOScoreRecord(new AXOL1TLScoreBxCollection());
```

Comment:
Why is the ```if (m_produceAXOL1TLScore)``` commented out? It looks like the object ```uGtAXOScoreRecord``` is later used only if ```m_produceAXOL1TLScore``` is actually true, so the object creation can also be within the if block.

Reply:
If I leave the declaration of uGtAXOScoreRecord within the if statement block I get` error: 'uGtAXOScoreRecord' was not declared in this scope` in the later if blocks in the code :

`src/L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc:748:26: error: 'uGtAXOScoreRecord' was not declared in this scope
  748 |     iEvent.put(std::move(uGtAXOScoreRecord), "AXOScore");`

Reply:
Did you try this?

```
    std::unique_ptr<AXOL1TLScoreBxCollection> uGtAXOScoreRecord(nullptr);
    if (m_produceAXOL1TLScore) {
        uGtAXOScoreRecord = std::make_unique<AXOL1TLScoreBxCollection>();
    }
```

Reply:
ok, I can implement it this way